src/app/events/event-details/event-details.component.html

```
@@ -19,20 +19,31 @@
-          mat-icon-button
-          [matMenuTriggerFor]="menu"
-          aria-label="Open event actions"
-        >
-          <fa-icon [icon]="faEllipsisVertical" />
-        </button>
-        <mat-menu #menu="matMenu">
-          <button mat-menu-item (click)="updateVisibility()">
-            Update listing
+        @if (eventQuery.isSuccess()) {
+          <button
+            title="menu"
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)